### PR TITLE
Microsoft Lync 2011 client

### PR DESCRIPTION
**Pull request by zzamboni**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
It’s missing `uninstall`. There are [scripts](https://github.com/phinze/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/phinze/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that. Do you need any help?

**Comment by zzamboni** [comment]
Thanks for the feedback. I've updated the PR with `uninstall`.

File: Casks/microsoft-lync.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 'da14ab83303717fdd01d5262be8cf4c83481b3188314d6527540ea3596bc38fe'
+
+  url "http://download.microsoft.com/download/5/0/0/500C7E1F-3235-47D4-BC11-95A71A1BA3ED/lync_#{version}.dmg"
+  name 'Microsoft Lync 2011'
+  homepage 'http://www.microsoft.com/mac/enterprise/lync'
+  license :unknown
+
+  pkg 'Lync Installer.pkg'
+
+  uninstall :pkgutil => [ 'com\.microsoft\.lync\.all\..*', 'com.microsoft.merp.all.errorreporting.pkg.2.2.9', 'com.microsoft.mau.all.autoupdate.pkg.2.3.6' ]
```
Comment (vitorgalvao) [inline Casks/microsoft-lync.rb]: There’s no need to escape the `.`. We also prefer arrays to be multi-line, [like this](https://github.com/caskroom/homebrew-cask/blob/62cd4871a9e2685064047e1c705dacf832f1c8a5/Casks/owncloud.rb#L12-15).

Comment (zzamboni) [inline Casks/microsoft-lync.rb]: Updated license and list formatting.

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by vitorgalvao** [comment]
Now that I look into it further, this seems to be an older version of the Lync plugin, for which we [already have a cask](https://github.com/caskroom/homebrew-cask/blob/master/Casks/microsoft-lync-plugin.rb). Is that the case? Why add this one?

**Comment by jawshooah** [comment]
The plugin is for the [web app](https://join.microsoft.com/meet/sebros/TWFQ3LYH), although it doesn't seem to be working on my machine. This cask is for the desktop client.

**Comment by vitorgalvao** [comment]
Ah, got it. Thank you.

**Comment by zzamboni** [comment]
Indeed, what Josh said :)

@jawshooah The web plugin has been hit-and-miss for me. I can usually use it to join meetings, but I have been unable to share my screen or a presentation with it. Sometimes it crashes and disconnects randomly.

**Pull request merged**